### ISSUE TITLE
Don't use *move* in `Pin` documentaton

### ISSUE THREAD

**Issue by dev-ardi** [rust-lang/rust#123135]
### Location

https://doc.rust-lang.org/std/pin/index.html#what-is-moving

### Summary

As explained there *move* usually has a different meaning in Rust. I suggest using another word to avoid this confusion.
We could use 
- Displace
- Transfer
- Relocate

**Comment by workingjubilee** [comment]
Huh?

**Comment by compiler-errors** [comment]
I do feel like this is weird:

> For the purposes of this module’s documentation, however, when we write *move* in italics, we mean specifically that the value has *moved* in the mechanical sense of being located at a new place in memory.

Of these options, the only one that makes sense is perhaps *relocate*.

**Comment by krtab** [comment]
I wonder if on top of being a hindrance to understanding by most, it would be show-stopper for people relying on a screen reader, or whose vision is otherwise impaired.

I don't have a strong opinion regarding the chosen term, but as a native speaker of a latin tongue (French) both "relocated" and "displaced" seem clear.

**Comment by workingjubilee** [comment]
> I wonder if on top of being a hindrance to understanding by most, it would be show-stopper for people relying on a screen reader, or whose vision is otherwise impaired.

No. I mean, depends on the screen-reader, but the words are HTML-tagged. They should be responsive to `<em>` and `<strong>` and are off-spec if they aren't.

**Comment by compiler-errors** [comment]
I still agree that we don't necessarily need to overload verbs with rust-specific meanings just for convenience in these docs, and italicization is a pretty small thing to distinguish `MOVE.for-the-purposes-of-borrowck` and `MOVE.relocate-bytes-in-memory`.

**Comment by workingjubilee** [comment]
Yeah, I think I saw it as a refinement of the usual sense but I can see how it can be confusing, having given it more thought.

I don't think simply swapping words is quite what we want, either, though.

**Comment by dev-ardi** [comment]
I think that it's also helpful to say somewhere that a semantic move usually means a bitwise move too
